README

```diff
@@ -40,8 +40,9 @@ GNU versions of these tools are recommended.
 
 Installation
 ------------
-Installing Geany is done by the following three commands:
-$ ./configure
+Installing Geany is done by the following commands:
+$ libtoolize
```

Comment:
It's called from inside `autogen.sh`, so yeah it shouldn't be needed to called explicitly.